frontend/revive-gallery-app/src/components/products/ProductDetails.js

```diff
@@ -28,33 +31,48 @@ const ProductDetails = () => {
         }
     })
     .then((data) => {
-        setProduct(data.products[0]);
+        console.log('Received data:', data.products[0]);
+        setProduct(data.products[0])
     })
     .catch((error) => console.error("Error fetching products:", error));
-  }, [productId]);
+  }, []);
 
-  if (Object.keys(product).length === 0) {
-    // if product is empty, render nothing
-    return null;
-  }
+  const productDemo = {
+    title: 'Sample Product',
+    description: 'This is a test product description.',
+    images: [watch1, watch2, watch3, watch4, watch5],
+  };
+ 
+  const categories = [
+    { id: 1, name: 'Category A' },
+    { id: 2, name: 'Category B' },
+  ];
 
```

Comment:
@bkarakala we can remove this hardcoded data as we are using real time data from db.